test/fixtures/terraform-plan-no-error/main.tf

```diff
@@ -0,0 +1,12 @@
+provider "digitalocean" {
+  token = "hello"
+}
```

Comment:
We don't have any DigitalOcean creds set up in our CI system. Would it make more sense to use `null_resource` here? After all, we just want to check that the `plan` output is correct, not that it works with any particular provider.

Reply:
Changed